src/main/resources/userEmailTemplateFragment-feedbackSessionResendAllLinks.html

```diff
@@ -0,0 +1,10 @@
+<br>
+<br>&nbsp;&nbsp; <strong>Feedback Session Name:</strong> ${feedbackSessionName}
+<br>&nbsp;&nbsp; <strong>Deadline:</strong> ${deadline}
+<br>
+To submit feedback for the above session, please go to this Web address: <a href="${submitUrl}">${submitUrl}</a>
+<br>
+<br>
+To view the responses for this session, please go to this Web address: <a href="${reportUrl}">${reportUrl}</a>
```

Comment:
(applies to both `submitUrl` and `reportUrl`) Here, if either one of the URLs are things like `(Feedback session is closed)`, then the anchor element will have that as the `href` attribute as well, which is not good because it leads nowhere. The wrapping anchor element should be moved to the Java code as well.